src/app/client/src/app/modules/core/services/user/user.service.ts

```diff
@@ -206,6 +206,9 @@ export class UserService {
     if (profileData.rootOrgId) {
       organisationIds.push(profileData.rootOrgId);
     }
+    if (profileData.skills === null) {
```

Comment:
This can be  `profileData.skills = _.get(profileData, 'skills' ) || []`

Reply:
Requested changes Done.